e2e/evm_methods_test.go

```diff
@@ -40,181 +40,135 @@ func runAllMethods() []jsonrpc.Method {
 	methodDefinition struct {
 		methodConfig
 		methodSuccessRates
 	}
 
-	// methodConfig contains the configuration for a method to be tested.
-	// This includes the total number of requests to send & the requests per second.
+	// methodConfig specifies the configuration for a method to be tested.
+	// - totalRequests: Total number of requests to send
+	// - rps:           Requests per second
 	methodConfig struct {
-		totalRequests int // Total number of requests to send
-		rps           int // Requests per second
+		totalRequests int
+		rps           int
 	}
 
 	// methodSuccessRates contains the minimum success rate and maximum
 	// latency requirements for a method to pass the load test.
+	// - successRate:   Minimum success rate (0-1)
+	// - maxP50Latency: Maximum P50 latency
+	// - maxP95Latency: Maximum P95 latency
+	// - maxP99Latency: Maximum P99 latency
 	methodSuccessRates struct {
-		successRate   float64       // Minimum success rate (0-1)
-		maxP50Latency time.Duration // Maximum P50 latency
-		maxP95Latency time.Duration // Maximum P95 latency
-		maxP99Latency time.Duration // Maximum P99 latency
+		successRate   float64
+		maxP50Latency time.Duration
+		maxP95Latency time.Duration
+		maxP99Latency time.Duration
 	}
 )
 
-// TODO_FUTURE(@commoddity): allow reading this configuration from a YAML file
+var (
+	// defaultMethodConfig contains the default configuration for a method.
+	defaultMethodConfig = methodConfig{
+		totalRequests: 100,
+		rps:           10,
+	}
+
+	// defaultMethodSuccessRates contains the default success rates and latency requirements for a method.
+	defaultMethodSuccessRates = methodSuccessRates{
+		successRate:   0.95,
+		maxP50Latency: 1_000 * time.Millisecond, //
+		maxP95Latency: 5_000 * time.Millisecond,
+		maxP99Latency: 10_000 * time.Millisecond,
+	}
+)
+
+// TODO_IMPROVE(@commoddity): allow reading this configuration from a YAML file
 //
 // methodDefinitions contains all method definitions for a service load test.
-// this allows customizing the configuration for each method as desired.
+// Allows customizing the configuration for each method as desired.
 var methodDefinitions = map[jsonrpc.Method]methodDefinition{
 	eth_blockNumber: {
-		methodConfig: methodConfig{
-			totalRequests: 200,
-			rps:           10,
-		},
-		methodSuccessRates: methodSuccessRates{
-			successRate:   0.95,
-			maxP50Latency: 350 * time.Millisecond,
-			maxP95Latency: 900 * time.Millisecond,
-			maxP99Latency: 3_000 * time.Millisecond,
-		},
+		methodConfig:       defaultMethodConfig,
+		methodSuccessRates: defaultMethodSuccessRates,
 	},
 	eth_call: {
-		methodConfig: methodConfig{
-			totalRequests: 200,
-			rps:           10,
-		},
-		methodSuccessRates: methodSuccessRates{
-			successRate:   0.90,
-			maxP50Latency: 350 * time.Millisecond,
-			maxP95Latency: 900 * time.Millisecond,
-			maxP99Latency: 3_000 * time.Millisecond,
-		},
+		methodConfig:       defaultMethodConfig,
```

Comment:
Using a default is a good improvement but previously I had set a lower expected success rate for `eth_call` specifically as it seems to fail more often than a lot of other requests. IF it's passing with <95% consistently in your experience then feel free to disregard.